pkg/backup/restore/restore_manager.go

```diff
@@ -1287,6 +1288,18 @@ func (rm *restoreManager) waitWarmUpJobsFinished(r *v1alpha1.Restore) error {
 				return err
 			} else if condition.Type == batchv1.JobComplete {
 				jobFinished = true
+				if r.Spec.WarmupStrategy == v1alpha1.RestoreWarmupStrategyCheckOnly {
```

Comment:
Here only indicates one tikv warmup finished, are you sure to update status here?